From 658dca06de0565c4702e095ab55b54cfb57e06b6 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Fri, 1 Jul 2011 10:28:21 +0200
Subject: [PATCH] crypt/cryptroot-ask.sh: s/.console.lock/.console_lock/g

In the initrd, the init script and the sbin/cryptroot-ask script use
flock with different lock files for the console: /.console_lock
and /.console.lock respectively
---
 modules.d/90crypt/cryptroot-ask.sh |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/modules.d/90crypt/cryptroot-ask.sh b/modules.d/90crypt/cryptroot-ask.sh
index 4a65c23..7b2ab07 100755
--- a/modules.d/90crypt/cryptroot-ask.sh
+++ b/modules.d/90crypt/cryptroot-ask.sh
@@ -94,7 +94,7 @@ else
             /bin/plymouth ask-for-password \
                 --prompt "$prompt" --number-of-tries=5 \
                 --command="/sbin/cryptsetup luksOpen -T1 $device $luksname"
-        } 9>/.console.lock
+        } 9>/.console_lock
         
         unset sluksname prompt
         
@@ -103,7 +103,7 @@ else
         { flock -s 9;
             echo "$device ($luksname) is password protected"
             cryptsetup luksOpen -T5 $device $luksname
-        } 9>/.console.lock
+        } 9>/.console_lock
     fi
 fi
 
-- 
1.7.9.3

